Home work vetka

andr777 opened the pull request:
задачи 1, 2, 3

andr777 committed:
-

andr777 committed:
задачи 1, 2, 3

iStemmer commented on src/Main.java:
```diff
@@ -1,5 +1,131 @@
+        //year = 2021;
+        year = LocalDate.now().getYear();
+
+        System.out.println(visokosnyiGod(year)); // visokosnyiGod() метод для определения високосности года, см.ниже.
```
переименовать в isLeapYear
Настоятельно рекомендую пользоваться переводчиком и не пользоваться транслитом.

andr777 replied:
сделал

iStemmer commented on src/Main.java:
```diff
@@ -1,5 +1,131 @@
+        int clientDeviceYear = 2022;
+        int typeOS = 0;
+
+        System.out.println(metodOpredeleniyaOS(typeOS, clientDeviceYear)); // metodOpredeleniyaOS() метод для определения что ставить
```
checkOs
методы мы обычно называем с глаголом - проверить, провалидировать, найти, достать, получить 
Так их и надо переводить. в данном случае - проверить версию ОС ----> checkVersionOs

andr777 replied:
сделал

andr777 committed:
задачи 1, 2, 3
замена названия методов
visokosnyiGod -> isLeapYear
metodOpredeleniyaOS -> checkVersionOs
